### ISSUE TITLE
Apply SIMBA for small dataset

### ISSUE THREAD

**Issue by hikahika12** [huidongchen/simba#13]
Hello!

I'm interested in applying SIMBA to a small dataset. Although I'm attempting to analyze scRNA-seq data with 15 cell nodes and 5000 gene nodes ("n_edges": 76947), the learning process doesn't seem to progress (attached image), and I'm not achieving good embedding results. Could this be due to the small data size?

Is there a limit to the data size to which SIMBA can be applied?

Thank you so much!
[pbg_metrics.pdf](https://github.com/huidongchen/simba/files/12544689/pbg_metrics.pdf)

**Comment by huidongchen** [comment]
It is clear that there's an issue of overfitting here. In a well-performing model, you should observe a **decrease in both training loss and validation loss** and an **increase in MRR** until they stabilize. 

What you can do is to further increase the weighty decay parameter, `wd`, until the model no longer exhibits overfitting.

e.g.,
```
dict_config = si.settings.pbg_params.copy()
dict_config['wd'] = new_larger_value
si.tl.pbg_train(pbg_params = dict_config, auto_wd=False, save_wd=True, output='model2')
```

**Comment by hikahika12** [comment]
I appreciate your quick response! 
I will try.
